generators/src/new-test-model/templates/model/types.tpl

```diff
@@ -0,0 +1,27 @@
+import {
+  {{modelName}},
+} from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type T{{modelName}}Rest = {{modelName}};
+
+// TODO: Check wether the graphql version has different properties
+// or types than the rest version.
+// You might want to use the GraphQL Explorer for that
+// (https://mc.europe-west1.gcp.escemo.com/test-with-big-data-44/graphql-explorer-commercetools/platform)
```

Comment:
Is this a url we want in a public repo?

Reply:
That's a good question.

I think the URL will help developers writing new models but I can see the conflict, specially since it's an _internal_ (Staging) one.

Maybe we can use a production (public) URL or just remove it and hope every engineer knows how to access GraphQL Explorer in the Merchant Center.

@tdeekens What's your take on this one? 🤔

Reply:
We tried to not expose staging URLs to the public. It's not cause there is something fundamentally risky in doing so. It's just that we try not to so there is no external expectation towards them. In this case we should just use a production URL or a placeholder.

Reply:
Thanks! 👍 

I finally changed it for a placeholder URL:
```
https://<merchant_center_domain>/<project_key>/graphql-explorer-commercetools/platform
```

Updated here: 681e14d45fbee868474df456b29fff910ec97d51